wolfcrypt/src/misc.c

```diff
@@ -39,11 +39,18 @@
  */
 
 #ifdef NO_INLINE
+    #undef  WOLFSSL_MISC_COMPILED
```

Comment:
I'm not sure it makes sense to define this when the user includes the file directly.  We're trying to distinguish the case of user explicit compile vs implicit header include compile, not make it more cloudy.